apps/omg_utils/lib/omg_utils/http_rpc/validators/base.ex

```diff
@@ -106,6 +106,10 @@ defmodule OMG.Utils.HttpRPC.Validator.Base do
   def optional({val, _}) when val in [:missing, nil], do: {nil, []}
   def optional(acc), do: acc
 
+  @spec optional({any(), list()}, atom()) :: {any(), list()}
```

Comment:
Nice - and kudos you were able to extent it - one of the most magical code :D :tada: